pkg/users/accounts-list.js

```diff
@@ -61,39 +61,30 @@ const UserActions = ({ account }) => {
     ];
 
     superuser.allowed && actions.push(
-        <DropdownSeparator key="separator-0" />,
+        <Divider key="separator-0" />,
         <DropdownItem key="log-user-out"
                       isDisabled={account.uid === 0 || !account.loggedIn}
-                      onClick={() => { setKebabOpen(false); logoutAccountDialog(account) }}>
+                      onClick={() => logoutAccountDialog(account) }>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20117-20240301-104857-b5f30935-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)